test/integration_test/common_test.go

```diff
@@ -816,21 +817,74 @@ func scheduleBidirectionalClusterPair(cpName, cpNamespace, projectMappings strin
 	// Create source --> destination and destination --> cluster pairs using storkctl
 	factory := storkctl.NewFactory()
 	cmd := storkctl.NewCommand(factory, os.Stdin, os.Stdout, os.Stderr)
-	cmd.SetArgs([]string{"create", "clusterpair", "-n", cpNamespace, cpName,
+	cmdArgs := []string{"create", "clusterpair", "-n", cpNamespace, cpName,
 		"--src-kube-file", srcKubeconfigPath,
-		"--src-ip", srcInfo[clusterIP],
-		"--src-token", srcInfo[tokenKey],
+		"--src-ep", fmt.Sprintf("%s:%s", srcInfo[clusterIP], srcInfo[clusterPort]),
 		"--dest-kube-file", destKubeconfigPath,
-		"--dest-ip", destInfo[clusterIP],
-		"--dest-token", destInfo[tokenKey],
-		"--project-mappings", projectMappings,
-	})
+		"--dest-ep", fmt.Sprintf("%s:%s", destInfo[clusterIP], destInfo[clusterPort]),
+	}
+
+	if projectMappings != "" {
+		cmdArgs = append(cmdArgs, fmt.Sprintf("--projectMappings %s", projectMappings))
+	}
 
+	// Get external object store details and append to the command accordingily
+	objectStoreArgs, err := getObjectstoreArgs(objectStoreType, secretName)
```

Comment:
nit..You might want to capitalize S in store in the name of the fun

Reply:
Fixed.